homeassistant/components/zha/core/gateway.py

```diff
@@ -36,6 +36,7 @@
 
 SENSOR_TYPES = {}
 BINARY_SENSOR_TYPES = {}
```

Comment:
That is not relevant for this PR, but globals are not allowed. Please migrate this in a future PR to `hass.data` or as a member inside gateway object.